ui/src/server/auth.ts

```diff
@@ -38,16 +40,27 @@ declare module "next-auth" {
  */
 export const authOptions: NextAuthOptions = {
   callbacks: {
-    session: ({ session, user }) => ({
+    session: ({ session, token }) => ({
```

Comment:
I think we also need the iser obj in addition

Reply:
Can you elaborate?
I followed the code here:
https://create.t3.gg/en/usage/next-auth#usage-with-nextjs-middleware

Reply:
You don't need to remove the user obj
Just do this:

```javascript
callbacks: {
     session: ({ session, user }) => ({
     session: ({ session, token }) => ({
        ...session,
        user: {
          ...session.user,
         id: user.id,
         id: token.sub,
        },
      }),
```

Reply:
Hmm on that case ia looks great